src/components/Editor/Tabs.js

```diff
@@ -12,7 +12,9 @@ import { getFilename, isPretty } from "../../utils/source";
 import classnames from "classnames";
 import actions from "../../actions";
 import CloseButton from "../shared/Button/Close";
-const PaneToggleButton = createFactory(require("../shared/Button/PaneToggle"));
+const PaneToggleButton = createFactory(
```

Comment:
What if we did:

```js
import _PaneToggleButton from "../shared/Button/Close";
const PaneToggleButton = createFactory(_PaneToggleButton)
```

this way we could separate concerns and still use import, and then createFactory below